docs/ref/checks.txt

```diff
@@ -99,6 +99,13 @@ Some checks may be registered with multiple tags.
 Core system checks
 ==================
 
+Asynchronous support
+--------------------
+
```

Comment:
This section is new, so `.. versionadded:: 3.1`.

Reply:
Okay